go.mod

```diff
@@ -1,8 +1,15 @@
 module github.com/goware/breaker
 
-go 1.16
+go 1.21
 
 require (
 	github.com/goware/superr v0.0.2
 	github.com/stretchr/testify v1.8.1
 )
+
+require (
+	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/pmezard/go-difflib v1.0.0 // indirect
+	github.com/stretchr/objx v0.5.0 // indirect
+	gopkg.in/yaml.v3 v3.0.1 // indirect
```

Comment:
where do these come from btw?

Reply:
ok, fair.. indirect deps were probably added in between go 1.16 -> go 1.21

Reply:
`go mod graph`

```
> go mod graph
github.com/goware/breaker github.com/davecgh/go-spew@v1.1.1
github.com/goware/breaker github.com/goware/superr@v0.0.2
github.com/goware/breaker github.com/pmezard/go-difflib@v1.0.0
github.com/goware/breaker github.com/stretchr/objx@v0.5.0
github.com/goware/breaker github.com/stretchr/testify@v1.8.1
github.com/goware/breaker go@1.21
github.com/goware/breaker gopkg.in/yaml.v3@v3.0.1
github.com/stretchr/objx@v0.5.0 github.com/stretchr/testify@v1.8.0
github.com/stretchr/testify@v1.8.1 github.com/davecgh/go-spew@v1.1.1
github.com/stretchr/testify@v1.8.1 github.com/pmezard/go-difflib@v1.0.0
github.com/stretchr/testify@v1.8.1 github.com/stretchr/objx@v0.5.0
github.com/stretchr/testify@v1.8.1 gopkg.in/yaml.v3@v3.0.1
go@1.21 toolchain@go1.21
gopkg.in/yaml.v3@v3.0.1 gopkg.in/check.v1@v0.0.0-20161208181325-20d25e280405
github.com/stretchr/testify@v1.8.0 github.com/davecgh/go-spew@v1.1.1
github.com/stretchr/testify@v1.8.0 github.com/pmezard/go-difflib@v1.0.0
github.com/stretchr/testify@v1.8.0 github.com/stretchr/objx@v0.4.0
github.com/stretchr/testify@v1.8.0 gopkg.in/yaml.v3@v3.0.1
github.com/stretchr/objx@v0.4.0 github.com/davecgh/go-spew@v1.1.1
github.com/stretchr/objx@v0.4.0 github.com/stretchr/testify@v1.7.1
github.com/stretchr/testify@v1.7.1 github.com/davecgh/go-spew@v1.1.0
github.com/stretchr/testify@v1.7.1 github.com/pmezard/go-difflib@v1.0.0
github.com/stretchr/testify@v1.7.1 github.com/stretchr/objx@v0.1.0
github.com/stretchr/testify@v1.7.1 gopkg.in/yaml.v3@v3.0.0-20200313102051-9f266ea9e77c
gopkg.in/yaml.v3@v3.0.0-20200313102051-9f266ea9e77c gopkg.in/check.v1@v0.0.0-20161208181325-20d25e280405
```